src/main/java/com/github/tomakehurst/wiremock/store/files/FileSourceBlobStore.java

```diff
@@ -56,7 +63,12 @@ public Stream<String> getAllKeys() {
 
   @Override
   public Optional<byte[]> get(String key) {
-    return Optional.of(fileSource.getBinaryFileNamed(key).readContents());
+    try {
+      return Optional.of(fileSource.getBinaryFileNamed(key).readContents());
+    } catch (Exception exception) {
```

Comment:
How about we catch `FileNotFoundException` and don't log anything, since this is a normal event, then log at error for any other type of exception (which in practice would probably indicate a misconfigured file system)?

Reply:
Done. I used instanceof because FileNotFoundException is cast to Throwable in the BinaryFile class via the throwUnchecked method